### PR TITLE
DAOS-6961-test: Pool rebuild test scripts update work around for DAOS-6865

### PR DESCRIPTION
**Pull request by dinghwah**
DAOS-6961-test: Pool rebuild test scripts update work around for DAOS-6865
Updated: pool/rebuild_tests.py
         pool/rebuild_tests.yaml
Skip-unit-tests: true
Skip-nlt: true
Test-tag: rebuildmulti rebuildsimple
Signed-off-by: Ding Ho <ding-hwa.ho@intel.com>

### CODE REVIEW

**Review by shimizukko — DISMISSED** [review]
LGTM

File: src/tests/ftest/pool/rebuild_tests.py [hunk 1]
```diff
@@ -16,6 +17,8 @@ class RebuildTests(TestWithServers):
     :avocado: recursive
     """
 
+    CANCEL_FOR_TICKET = [["DAOS-6983", "object_qty", 20]]
```
[the remark this answers is not in the document]
Comment (dinghwah) [reply src/tests/ftest/pool/rebuild_tests.py:20]: DAOS-6983 is a new bug on obj. counter after the rebuild.

File: src/tests/ftest/pool/rebuild_tests.py [hunk 2]
```diff
@@ -81,8 +94,9 @@ def run_rebuild_test(self, pool_quantity):
                 self.pool[index].exclude([rank], self.d_log)
 
         # Wait for recovery to start
-        for index in range(pool_quantity):
-            self.pool[index].wait_for_rebuild(True)
+        if pool_quantity < 2:
+            for index in range(pool_quantity):
```
[the remark this answers is not in the document]
Comment (dinghwah) [reply src/tests/ftest/pool/rebuild_tests.py:98]: Please see the new commit.

**Review by shimizukko — APPROVED** [review]
LGTM

**Review by saurabhtandan — COMMENTED** [review]
There are 10 failures in the CI results. If there is another issue for those, lets identify it open a ticket, skip those and let's get the 100 passing tests in CI.
Also, I noticed there are two test methods in this test class, `test_simple_rebuild` and `test_multipool_rebuild` and both run on daily_regression tag and account for 2 hr 43 min of time. This can be considerable amount of time addition for daily regression. How about we let `test_simple_rebuild` run as part of daily regression and run `test_multipool_rebuild` as part of weekly?
May be something we can discuss on rebuild WG.

**Pull request closed without merging**